pkg/apis/config/v1alpha1/types.go

```diff
@@ -31,12 +31,6 @@ type ControllerConfiguration struct {
 	// ETCD is the etcd configuration.
 	ETCD ETCD `json:"etcd"`
 
-	// ClusterAudit is the configuration for cluster auditing.
-	ClusterAudit ClusterAudit `json:"clusterAudit"`
```

Comment:
Probably we can not remove the fields from the API right now because then old fields will still be contained in some shoot specs and then it will complain about unknown fields.

Reply:
I kept the API fields now.